cmd/main.go

```diff
@@ -41,12 +41,14 @@ func main() {
 		Style: pterm.NewStyle(pterm.FgRed),
 	}
 
+	pterm.Warning.Prefix = pterm.Prefix{
+		Text:  "⚠",
```

Comment:
This looks great, but I'm not sure if this is a right place for this :trollface: but I don't know where is the right place, so let's keep it here.

Reply:
![](https://github.githubassets.com/images/icons/emoji/trollface.png)